Documentation/policy/lifecycle.rst

```diff
@@ -45,14 +45,19 @@ a change in identity, policy, or configuration.
 An endpoint transitions into the ``disconnecting`` state when it is
 being deleted, regardless of its current state.
 
-In some environments, notably Docker and Kubernetes, Cilium can't
-determine the labels of an endpoint immediately when the endpoint is
-created, and therefore can't allocate an identity for the endpoint at
-that point.  Until the endpoint's labels are known, Cilium temporarily
-associates a special single label ``reserved:init`` to the endpoint.
-When the endpoint's labels become known, Cilium then replaces that
-special label with the endpoint's labels and allocates a proper
-identity to the endpoint.
+.. _init_identity:
+
+Init Identity
+-------------
+
+In some environments, notably Docker via libnetwork or Kubernetes when the API
```

Comment:
IIRC this can also happen if Cilium isn't able to derive the security identity of the set of labels. For example, if etcd is not reachable.